src/mappings/handlers/multisigRemarkHandler.ts

```diff
@@ -0,0 +1,56 @@
+import { SubstrateEvent } from "@subql/types";
+import { checkAndGetAccount } from "../../utils/checkAndGetAccount";
+import { checkAndGetAccountMultisig } from "../../utils/checkAndGetAccountMultisig";
+import { decodeAddress, createKeyMultiAddress } from "../../utils";
+import { u8aToHex } from "@polkadot/util";
+import { MultisigRemarkArgs } from "../types";
+
+export async function handleMultisigRemarkEventHandler(event: SubstrateEvent): Promise<void> {
+  if (!event) return;
+
+  const extrinsic = event.extrinsic?.extrinsic;
+
+  if (!extrinsic) return;
+
+  const args = extrinsic.args[0]?.toHuman() as unknown as string;
+
+  let parsedArgs: MultisigRemarkArgs;
+  try {
+    parsedArgs = JSON.parse(args) as unknown as MultisigRemarkArgs;
+  } catch (e) {
+    return;
+  }
+
+  if (!parsedArgs || !parsedArgs.signatories || !parsedArgs.threshold) return;
```

Comment:
it would be nice to add a debug logger after this line, to see which remark was parsed and how multisigs are created.